src/main/java/racing/domain/Car.java

```diff
@@ -4,24 +4,35 @@ public class Car {
     private final String name;
     private int distance;
 
-    Car() {
-        this.name = "";
-        this.distance = 0;
+    Car(String name) {
+        this(name, 0);
     }
 
-    private Car(int distance) {
-        this.name = "";
+    private Car(String name, int distance) {
+        this.name = name;
         this.distance = distance;
     }
 
     Car move(Movable movable) {
         if (movable.isMovable()) {
-            return new Car(++distance);
+            return new Car(name, ++distance);
         }
-        return new Car(distance);
+        return new Car(name, distance);
+    }
+
+    int getGreaterDistance(int distance) {
+        return this.distance > distance ? this.distance : distance;
```

Comment:
`Math.max()`를 활용해보면 어떨까요?
```suggestion
        return Math.max(this.distance, distance);
```

Reply:
이런 메소드가 있다는 건 어떻게 아셨나요...?😮‍💨

Reply:
혹시 기본서 중에서 `자바의 정석`이 있으시다면 9장을 참고하시는 걸 추천드립니다. [동영상](https://www.youtube.com/watch?v=NI6QZy6juc8&ab_channel=%EB%82%A8%EA%B6%81%EC%84%B1%EC%9D%98%EC%A0%95%EC%84%9D%EC%BD%94%EB%94%A9)